### ISSUE TITLE
Hide heading level tag for chart

### ISSUE THREAD

**Issue by RubyZ10** [highcharts/highcharts#17802]
#### Expected behaviour
Screen reader should not announce heading level of chart when there is no headings visible on the page for accessibility compliance.

#### Actual behaviour
Chart has default heading and screen readers announces heading level, even it is not visible on the page.
![image](https://user-images.githubusercontent.com/76190371/193161441-97b593d7-1c85-47d9-ad67-4cb669ae9ece.png)

**Comment by oysteinmoseng** [comment]
@RubyZ10 I agree that the default behavior here is debatable, although it will also make sense in some cases. For now you can override the default behavior using the accessibility API options, e.g. [accessibility.screenReaderSection.beforeChartFormat](https://api.highcharts.com/highcharts/accessibility.screenReaderSection.beforeChartFormat).

**Comment by RubyZ10** [comment]
Thanks @oysteinmoseng used beforeChartFormat to override!
